L 11/09/2019 - 00:04:31: SourceMod error session started
L 11/09/2019 - 00:04:31: Info (map "c2m4_barns") (file "errors_20191109.log")
L 11/09/2019 - 00:04:31: [SM] Exception reported: Client 3 is not in game
L 11/09/2019 - 00:04:31: [SM] Blaming: l4d2_tank_attack_control_nextmod.smx
L 11/09/2019 - 00:04:31: [SM] Call stack trace:
L 11/09/2019 - 00:04:31: [SM]   [0] IsPlayerAlive
L 11/09/2019 - 00:04:31: [SM]   [1] Line 220, C:\Users\Trevor_2\Desktop\New Desktop Mass Storage Lmfao\scripting\l4d2_tank_attack_control_nextmod.sp::Timer_Countdown
L 11/09/2019 - 00:04:32: [SM] Exception reported: Client 3 is not in game
L 11/09/2019 - 00:04:32: [SM] Blaming: l4d2_tank_attack_control_nextmod.smx
L 11/09/2019 - 00:04:32: [SM] Call stack trace:
L 11/09/2019 - 00:04:32: [SM]   [0] IsPlayerAlive
L 11/09/2019 - 00:04:32: [SM]   [1] Line 220, C:\Users\Trevor_2\Desktop\New Desktop Mass Storage Lmfao\scripting\l4d2_tank_attack_control_nextmod.sp::Timer_Countdown
L 11/09/2019 - 00:04:33: [SM] Exception reported: Client 3 is not in game
L 11/09/2019 - 00:04:33: [SM] Blaming: l4d2_tank_attack_control_nextmod.smx
L 11/09/2019 - 00:04:33: [SM] Call stack trace:
L 11/09/2019 - 00:04:33: [SM]   [0] IsPlayerAlive
L 11/09/2019 - 00:04:33: [SM]   [1] Line 220, C:\Users\Trevor_2\Desktop\New Desktop Mass Storage Lmfao\scripting\l4d2_tank_attack_control_nextmod.sp::Timer_Countdown
L 11/09/2019 - 00:04:34: [SM] Exception reported: Client 3 is not in game
L 11/09/2019 - 00:04:34: [SM] Blaming: l4d2_tank_attack_control_nextmod.smx
L 11/09/2019 - 00:04:34: [SM] Call stack trace:
L 11/09/2019 - 00:04:34: [SM]   [0] IsPlayerAlive
L 11/09/2019 - 00:04:34: [SM]   [1] Line 208, C:\Users\Trevor_2\Desktop\New Desktop Mass Storage Lmfao\scripting\l4d2_tank_attack_control_nextmod.sp::ResetJumpRockCooldown
L 11/09/2019 - 00:04:34: [SM] Exception reported: Client 3 is not in game
L 11/09/2019 - 00:04:34: [SM] Blaming: l4d2_tank_attack_control_nextmod.smx
L 11/09/2019 - 00:04:34: [SM] Call stack trace:
L 11/09/2019 - 00:04:34: [SM]   [0] IsPlayerAlive
L 11/09/2019 - 00:04:34: [SM]   [1] Line 220, C:\Users\Trevor_2\Desktop\New Desktop Mass Storage Lmfao\scripting\l4d2_tank_attack_control_nextmod.sp::Timer_Countdown
L 11/09/2019 - 00:04:35: [SM] Exception reported: Client 3 is not in game
L 11/09/2019 - 00:04:35: [SM] Blaming: l4d2_tank_attack_control_nextmod.smx
L 11/09/2019 - 00:04:35: [SM] Call stack trace:
L 11/09/2019 - 00:04:35: [SM]   [0] IsPlayerAlive
L 11/09/2019 - 00:04:35: [SM]   [1] Line 220, C:\Users\Trevor_2\Desktop\New Desktop Mass Storage Lmfao\scripting\l4d2_tank_attack_control_nextmod.sp::Timer_Countdown
L 11/09/2019 - 00:04:36: [SM] Exception reported: Client 3 is not in game
L 11/09/2019 - 00:04:36: [SM] Blaming: l4d2_tank_attack_control_nextmod.smx
L 11/09/2019 - 00:04:36: [SM] Call stack trace:
L 11/09/2019 - 00:04:36: [SM]   [0] IsPlayerAlive
L 11/09/2019 - 00:04:36: [SM]   [1] Line 220, C:\Users\Trevor_2\Desktop\New Desktop Mass Storage Lmfao\scripting\l4d2_tank_attack_control_nextmod.sp::Timer_Countdown
L 11/09/2019 - 00:04:37: [SM] Exception reported: Client 3 is not in game
L 11/09/2019 - 00:04:37: [SM] Blaming: l4d2_tank_attack_control_nextmod.smx
L 11/09/2019 - 00:04:37: [SM] Call stack trace:
L 11/09/2019 - 00:04:37: [SM]   [0] IsPlayerAlive
L 11/09/2019 - 00:04:37: [SM]   [1] Line 220, C:\Users\Trevor_2\Desktop\New Desktop Mass Storage Lmfao\scripting\l4d2_tank_attack_control_nextmod.sp::Timer_Countdown
L 11/09/2019 - 00:08:03: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 00:08:03: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 00:08:03: [SM] Call stack trace:
L 11/09/2019 - 00:08:03: [SM]   [0] SetEntProp
L 11/09/2019 - 00:08:03: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 00:10:18: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 00:10:18: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 00:10:18: [SM] Call stack trace:
L 11/09/2019 - 00:10:18: [SM]   [0] SetEntProp
L 11/09/2019 - 00:10:18: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 00:11:23: [SM] Exception reported: Client 10 is not in game
L 11/09/2019 - 00:11:23: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 00:11:23: [SM] Call stack trace:
L 11/09/2019 - 00:11:23: [SM]   [0] GetClientTeam
L 11/09/2019 - 00:11:23: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 00:11:24: Error log file session closed.
L 11/09/2019 - 00:16:13: SourceMod error session started
L 11/09/2019 - 00:16:13: Info (map "c2m5_concert") (file "errors_20191109.log")
L 11/09/2019 - 00:16:13: [SM] Exception reported: Entity 3 (3) is invalid
L 11/09/2019 - 00:16:13: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 00:16:13: [SM] Call stack trace:
L 11/09/2019 - 00:16:13: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 00:16:13: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 00:16:13: [SM] Exception reported: Entity 5 (5) is invalid
L 11/09/2019 - 00:16:13: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 00:16:13: [SM] Call stack trace:
L 11/09/2019 - 00:16:13: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 00:16:13: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 00:16:14: Error log file session closed.
L 11/09/2019 - 00:18:56: SourceMod error session started
L 11/09/2019 - 00:18:56: Info (map "c11m1_greenhouse") (file "errors_20191109.log")
L 11/09/2019 - 00:18:56: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 00:18:56: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 00:18:56: [SM] Call stack trace:
L 11/09/2019 - 00:18:56: [SM]   [0] SetEntProp
L 11/09/2019 - 00:18:56: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 00:19:39: [SM] Exception reported: Client index 153 is invalid
L 11/09/2019 - 00:19:39: [SM] Blaming: l4d2_getup_slide_fix.smx
L 11/09/2019 - 00:19:39: [SM] Call stack trace:
L 11/09/2019 - 00:19:39: [SM]   [0] IsClientInGame
L 11/09/2019 - 00:19:39: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 11/09/2019 - 00:19:39: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 11/09/2019 - 00:24:42: [SM] Exception reported: Entity 8 (8) is invalid
L 11/09/2019 - 00:24:42: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 00:24:42: [SM] Call stack trace:
L 11/09/2019 - 00:24:42: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 00:24:42: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 11/09/2019 - 00:24:43: Error log file session closed.
L 11/09/2019 - 00:35:30: SourceMod error session started
L 11/09/2019 - 00:35:30: Info (map "c11m2_offices") (file "errors_20191109.log")
L 11/09/2019 - 00:35:30: [SM] Exception reported: Entity 10 (10) is invalid
L 11/09/2019 - 00:35:30: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 00:35:30: [SM] Call stack trace:
L 11/09/2019 - 00:35:30: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 00:35:30: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 00:35:30: [SM] Exception reported: Entity 10 (10) is invalid
L 11/09/2019 - 00:35:30: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 00:35:30: [SM] Call stack trace:
L 11/09/2019 - 00:35:30: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 00:35:30: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 11/09/2019 - 00:43:05: [SM] Exception reported: Entity 6 (6) is invalid
L 11/09/2019 - 00:43:05: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 00:43:05: [SM] Call stack trace:
L 11/09/2019 - 00:43:05: [SM]   [0] GetEntProp
L 11/09/2019 - 00:43:05: [SM]   [1] Line 371, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::IsPlayingGetUpAnimation
L 11/09/2019 - 00:43:05: [SM]   [2] Line 211, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::BlueMoonCaseCheck
L 11/09/2019 - 00:44:35: Error log file session closed.
L 11/09/2019 - 00:53:46: SourceMod error session started
L 11/09/2019 - 00:53:46: Info (map "c11m3_garage") (file "errors_20191109.log")
L 11/09/2019 - 00:53:46: [SM] Exception reported: Client 6 is not in game
L 11/09/2019 - 00:53:46: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 00:53:46: [SM] Call stack trace:
L 11/09/2019 - 00:53:46: [SM]   [0] GetClientTeam
L 11/09/2019 - 00:53:46: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 00:56:52: [SM] Exception reported: Entity 8 (8) is invalid
L 11/09/2019 - 00:56:52: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 00:56:52: [SM] Call stack trace:
L 11/09/2019 - 00:56:52: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 00:56:52: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 11/09/2019 - 00:56:52: [SM] Exception reported: Client 6 is not in game
L 11/09/2019 - 00:56:52: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 00:56:52: [SM] Call stack trace:
L 11/09/2019 - 00:56:52: [SM]   [0] GetClientTeam
L 11/09/2019 - 00:56:52: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 00:56:55: Error log file session closed.
L 11/09/2019 - 00:59:17: SourceMod error session started
L 11/09/2019 - 00:59:17: Info (map "c11m4_terminal") (file "errors_20191109.log")
L 11/09/2019 - 00:59:17: [SM] Exception reported: Client index 105 is invalid
L 11/09/2019 - 00:59:17: [SM] Blaming: l4d2_getup_slide_fix.smx
L 11/09/2019 - 00:59:17: [SM] Call stack trace:
L 11/09/2019 - 00:59:17: [SM]   [0] IsClientInGame
L 11/09/2019 - 00:59:17: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 11/09/2019 - 00:59:17: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 11/09/2019 - 00:59:19: [SM] Exception reported: Client index 105 is invalid
L 11/09/2019 - 00:59:19: [SM] Blaming: l4d2_getup_slide_fix.smx
L 11/09/2019 - 00:59:19: [SM] Call stack trace:
L 11/09/2019 - 00:59:19: [SM]   [0] IsClientInGame
L 11/09/2019 - 00:59:19: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 11/09/2019 - 00:59:19: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 11/09/2019 - 01:06:34: [SM] Exception reported: Client index 190 is invalid
L 11/09/2019 - 01:06:34: [SM] Blaming: l4d2_getup_slide_fix.smx
L 11/09/2019 - 01:06:34: [SM] Call stack trace:
L 11/09/2019 - 01:06:34: [SM]   [0] IsClientInGame
L 11/09/2019 - 01:06:34: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 11/09/2019 - 01:06:34: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 11/09/2019 - 01:08:22: Error log file session closed.
L 11/09/2019 - 01:19:33: SourceMod error session started
L 11/09/2019 - 01:19:33: Info (map "c11m5_runway") (file "errors_20191109.log")
L 11/09/2019 - 01:19:33: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 01:19:33: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 01:19:33: [SM] Call stack trace:
L 11/09/2019 - 01:19:33: [SM]   [0] SetEntProp
L 11/09/2019 - 01:19:33: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 01:27:51: [SM] Exception reported: Client 6 is not in game
L 11/09/2019 - 01:27:51: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 01:27:51: [SM] Call stack trace:
L 11/09/2019 - 01:27:51: [SM]   [0] GetClientTeam
L 11/09/2019 - 01:27:51: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 01:27:51: [SM] Exception reported: Entity 6 (6) is invalid
L 11/09/2019 - 01:27:51: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 01:27:51: [SM] Call stack trace:
L 11/09/2019 - 01:27:51: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 01:27:51: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 01:27:51: [SM] Exception reported: Entity 1 (1) is invalid
L 11/09/2019 - 01:27:51: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 01:27:51: [SM] Call stack trace:
L 11/09/2019 - 01:27:51: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 01:27:51: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 01:27:54: Error log file session closed.
L 11/09/2019 - 01:36:18: SourceMod error session started
L 11/09/2019 - 01:36:18: Info (map "c2m1_highway") (file "errors_20191109.log")
L 11/09/2019 - 01:36:18: [SM] Exception reported: Entity 2 (2) is invalid
L 11/09/2019 - 01:36:18: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 01:36:18: [SM] Call stack trace:
L 11/09/2019 - 01:36:18: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 01:36:18: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 11/09/2019 - 01:36:34: Error log file session closed.
L 11/09/2019 - 01:39:52: SourceMod error session started
L 11/09/2019 - 01:39:52: Info (map "c2m2_fairgrounds") (file "errors_20191109.log")
L 11/09/2019 - 01:39:52: [SM] Exception reported: Client index 296 is invalid
L 11/09/2019 - 01:39:52: [SM] Blaming: l4d2_getup_slide_fix.smx
L 11/09/2019 - 01:39:52: [SM] Call stack trace:
L 11/09/2019 - 01:39:52: [SM]   [0] IsClientInGame
L 11/09/2019 - 01:39:52: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 11/09/2019 - 01:39:52: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 11/09/2019 - 01:48:14: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 01:48:14: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 01:48:14: [SM] Call stack trace:
L 11/09/2019 - 01:48:14: [SM]   [0] SetEntProp
L 11/09/2019 - 01:48:14: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 01:51:19: [SM] Exception reported: Entity 2 (2) is invalid
L 11/09/2019 - 01:51:19: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 01:51:19: [SM] Call stack trace:
L 11/09/2019 - 01:51:19: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 01:51:19: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 01:51:20: Error log file session closed.
L 11/09/2019 - 01:58:58: SourceMod error session started
L 11/09/2019 - 01:58:58: Info (map "c2m3_coaster") (file "errors_20191109.log")
L 11/09/2019 - 01:58:58: [SM] Exception reported: Client 7 is not in game
L 11/09/2019 - 01:58:58: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 01:58:58: [SM] Call stack trace:
L 11/09/2019 - 01:58:58: [SM]   [0] GetClientTeam
L 11/09/2019 - 01:58:58: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 01:58:59: Error log file session closed.
L 11/09/2019 - 02:02:58: SourceMod error session started
L 11/09/2019 - 02:02:58: Info (map "c2m4_barns") (file "errors_20191109.log")
L 11/09/2019 - 02:02:58: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 02:02:58: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 02:02:58: [SM] Call stack trace:
L 11/09/2019 - 02:02:58: [SM]   [0] SetEntProp
L 11/09/2019 - 02:02:58: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 02:08:51: [SM] Exception reported: Entity 9 (9) is invalid
L 11/09/2019 - 02:08:51: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 02:08:51: [SM] Call stack trace:
L 11/09/2019 - 02:08:51: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 02:08:51: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 11/09/2019 - 02:12:03: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 02:12:03: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 02:12:03: [SM] Call stack trace:
L 11/09/2019 - 02:12:03: [SM]   [0] SetEntProp
L 11/09/2019 - 02:12:03: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 02:12:15: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 02:12:15: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 02:12:15: [SM] Call stack trace:
L 11/09/2019 - 02:12:15: [SM]   [0] SetEntProp
L 11/09/2019 - 02:12:15: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 02:12:50: Error log file session closed.
L 11/09/2019 - 02:17:24: SourceMod error session started
L 11/09/2019 - 02:17:24: Info (map "c2m5_concert") (file "errors_20191109.log")
L 11/09/2019 - 02:17:24: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 02:17:24: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 02:17:24: [SM] Call stack trace:
L 11/09/2019 - 02:17:24: [SM]   [0] SetEntProp
L 11/09/2019 - 02:17:24: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 02:17:49: [SM] Exception reported: Client 6 is not in game
L 11/09/2019 - 02:17:49: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 02:17:49: [SM] Call stack trace:
L 11/09/2019 - 02:17:49: [SM]   [0] GetClientTeam
L 11/09/2019 - 02:17:49: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 02:17:52: Error log file session closed.
L 11/09/2019 - 02:27:07: SourceMod error session started
L 11/09/2019 - 02:27:07: Info (map "c3m1_plankcountry") (file "errors_20191109.log")
L 11/09/2019 - 02:27:07: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 02:27:07: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 02:27:07: [SM] Call stack trace:
L 11/09/2019 - 02:27:07: [SM]   [0] SetEntProp
L 11/09/2019 - 02:27:07: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 02:27:51: [SM] Exception reported: Entity 1 (1) is invalid
L 11/09/2019 - 02:27:51: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 02:27:51: [SM] Call stack trace:
L 11/09/2019 - 02:27:51: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 02:27:51: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 02:27:51: [SM] Exception reported: Entity 3 (3) is invalid
L 11/09/2019 - 02:27:51: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 02:27:51: [SM] Call stack trace:
L 11/09/2019 - 02:27:51: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 02:27:51: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 02:27:53: Error log file session closed.
L 11/09/2019 - 02:40:16: SourceMod error session started
L 11/09/2019 - 02:40:16: Info (map "c3m2_swamp") (file "errors_20191109.log")
L 11/09/2019 - 02:40:16: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 02:40:16: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 02:40:16: [SM] Call stack trace:
L 11/09/2019 - 02:40:16: [SM]   [0] SetEntProp
L 11/09/2019 - 02:40:16: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 02:40:44: Error log file session closed.
L 11/09/2019 - 02:44:32: SourceMod error session started
L 11/09/2019 - 02:44:32: Info (map "c3m3_shantytown") (file "errors_20191109.log")
L 11/09/2019 - 02:44:32: [SM] Exception reported: Client 3 is not in game
L 11/09/2019 - 02:44:32: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 02:44:32: [SM] Call stack trace:
L 11/09/2019 - 02:44:32: [SM]   [0] GetClientTeam
L 11/09/2019 - 02:44:32: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 02:45:19: [SM] Exception reported: Entity 1 (1) is invalid
L 11/09/2019 - 02:45:19: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 02:45:19: [SM] Call stack trace:
L 11/09/2019 - 02:45:19: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 02:45:19: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 11/09/2019 - 02:49:18: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 02:49:18: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 02:49:18: [SM] Call stack trace:
L 11/09/2019 - 02:49:18: [SM]   [0] SetEntProp
L 11/09/2019 - 02:49:18: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 02:49:21: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 02:49:21: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 02:49:21: [SM] Call stack trace:
L 11/09/2019 - 02:49:21: [SM]   [0] SetEntProp
L 11/09/2019 - 02:49:21: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 02:49:47: [SM] Exception reported: Entity 5 (5) is invalid
L 11/09/2019 - 02:49:47: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 02:49:47: [SM] Call stack trace:
L 11/09/2019 - 02:49:47: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 02:49:47: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 02:49:47: [SM] Exception reported: Entity 2 (2) is invalid
L 11/09/2019 - 02:49:47: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 02:49:47: [SM] Call stack trace:
L 11/09/2019 - 02:49:47: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 02:49:47: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 02:49:49: Error log file session closed.
L 11/09/2019 - 02:52:13: SourceMod error session started
L 11/09/2019 - 02:52:13: Info (map "c3m4_plantation") (file "errors_20191109.log")
L 11/09/2019 - 02:52:13: [SM] Exception reported: Client 1 is not in game
L 11/09/2019 - 02:52:13: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 02:52:13: [SM] Call stack trace:
L 11/09/2019 - 02:52:13: [SM]   [0] GetClientTeam
L 11/09/2019 - 02:52:13: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 03:00:59: Error log file session closed.
L 11/09/2019 - 03:04:05: SourceMod error session started
L 11/09/2019 - 03:04:05: Info (map "c1m1_hotel") (file "errors_20191109.log")
L 11/09/2019 - 03:04:05: [SM] Exception reported: Client index 391 is invalid
L 11/09/2019 - 03:04:05: [SM] Blaming: l4d2_getup_slide_fix.smx
L 11/09/2019 - 03:04:05: [SM] Call stack trace:
L 11/09/2019 - 03:04:05: [SM]   [0] IsClientInGame
L 11/09/2019 - 03:04:05: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 11/09/2019 - 03:04:05: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 11/09/2019 - 03:04:09: [SM] Exception reported: Client index 391 is invalid
L 11/09/2019 - 03:04:09: [SM] Blaming: l4d2_getup_slide_fix.smx
L 11/09/2019 - 03:04:09: [SM] Call stack trace:
L 11/09/2019 - 03:04:09: [SM]   [0] IsClientInGame
L 11/09/2019 - 03:04:09: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 11/09/2019 - 03:04:09: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 11/09/2019 - 03:08:20: Error log file session closed.
L 11/09/2019 - 03:08:49: SourceMod error session started
L 11/09/2019 - 03:08:49: Info (map "c1m2_streets") (file "errors_20191109.log")
L 11/09/2019 - 03:08:49: [SM] Exception reported: Client 8 is not in game
L 11/09/2019 - 03:08:49: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 03:08:49: [SM] Call stack trace:
L 11/09/2019 - 03:08:49: [SM]   [0] GetClientTeam
L 11/09/2019 - 03:08:49: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 03:23:44: [SM] Exception reported: Entity 7 (7) is invalid
L 11/09/2019 - 03:23:44: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 03:23:44: [SM] Call stack trace:
L 11/09/2019 - 03:23:44: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 03:23:44: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 03:24:11: [SM] Exception reported: Entity 3 (3) is invalid
L 11/09/2019 - 03:24:11: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 03:24:11: [SM] Call stack trace:
L 11/09/2019 - 03:24:11: [SM]   [0] GetEntDataEnt2
L 11/09/2019 - 03:24:11: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 03:24:13: Error log file session closed.
L 11/09/2019 - 03:25:51: SourceMod error session started
L 11/09/2019 - 03:25:51: Info (map "c1m3_mall") (file "errors_20191109.log")
L 11/09/2019 - 03:25:51: [SM] Exception reported: Client index 92 is invalid
L 11/09/2019 - 03:25:51: [SM] Blaming: l4d2_getup_slide_fix.smx
L 11/09/2019 - 03:25:51: [SM] Call stack trace:
L 11/09/2019 - 03:25:51: [SM]   [0] IsClientInGame
L 11/09/2019 - 03:25:51: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 11/09/2019 - 03:25:51: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 11/09/2019 - 03:26:42: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 03:26:42: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 03:26:42: [SM] Call stack trace:
L 11/09/2019 - 03:26:42: [SM]   [0] SetEntProp
L 11/09/2019 - 03:26:42: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 03:37:58: [SM] Exception reported: Entity -1 (-1) is invalid
L 11/09/2019 - 03:37:58: [SM] Blaming: l4d2_melee_shenanigans.smx
L 11/09/2019 - 03:37:58: [SM] Call stack trace:
L 11/09/2019 - 03:37:58: [SM]   [0] SetEntProp
L 11/09/2019 - 03:37:58: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 11/09/2019 - 03:42:36: Error log file session closed.
L 11/09/2019 - 03:44:23: SourceMod error session started
L 11/09/2019 - 03:44:23: Info (map "c1m4_atrium") (file "errors_20191109.log")
L 11/09/2019 - 03:44:23: [SM] Exception reported: Client 3 is not in game
L 11/09/2019 - 03:44:23: [SM] Blaming: l4d2_charger_getup_fix.smx
L 11/09/2019 - 03:44:23: [SM] Call stack trace:
L 11/09/2019 - 03:44:23: [SM]   [0] GetClientTeam
L 11/09/2019 - 03:44:23: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 11/09/2019 - 03:44:28: Error log file session closed.
